tfhe/src/high_level_api/tag.rs

```diff
@@ -0,0 +1,420 @@
+use crate::high_level_api::backward_compatibility::tag::TagVersions;
+use tfhe_versionable::{Unversionize, UnversionizeError, Versionize, VersionizeOwned};
+
+const STACK_ARRAY_SIZE: usize = std::mem::size_of::<Vec<u8>>() - 1;
```

Comment:
this may not be constant across compiler versions and perhaps even platforms/hw ?

could be an issue with WASM as well ?

Reply:
This would just mean that if for example sizeof::Vec is 3*32 = 96, then, a 128 bit number would be stored in a Vec rather than a the stack.

So when creating, no problem, when deserializing no problem either, and if a 64 bit CPU deserialize that, as it would fit on stack, the 128 bits would live on the stack

Reply:
all right I need to check the details of the small vec stuff then

Reply:
That's why it's important to convert it to a plain array before serialization (as you did)